reference-architecture/vmware-ansible/roles/openshift-auth-proxy-setup/tasks/main.yaml

```diff
@@ -0,0 +1,39 @@
+- name: Generate Certificate
+  command: "openshift start --public-master={{openshift_master_cluster_public_hostname}}:{{console_port}} --write-config=/etc/origin"
```

Comment:
Its not used at all. It was a remnant from my predecessor. I've removed the role completely. I do all the auth config in the openshift_install playbook.